modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/LastZ.pm

```diff
@@ -78,17 +56,17 @@ sub run {
   # get the sequences and create the runnable
   #
   my $query_DnaFragChunkSet = $self->param('query_DnaFragChunkSet');
-  my $qyChunkFile = $self->dumpChunkSetToWorkdir($query_DnaFragChunkSet);
+  my $qyChunkSetFile = $self->dumpChunkSetToTmp($query_DnaFragChunkSet);
 
+  my $dbChunkSet = $self->param('db_DnaFragChunkSet');
+  $dbChunkSet->load_all_sequences();
   my @db_chunk_files;
-  my $db_dna_collection = $self->param('db_DnaFragChunkSet')->dna_collection;
-  foreach my $db_chunk (@{$self->param('db_DnaFragChunkSet')->get_all_DnaFragChunks}) {
-      $db_chunk->masking($db_dna_collection->masking);
-    push @db_chunk_files, $self->dumpChunkToWorkdir($db_chunk, $db_dna_collection);
+  foreach my $db_chunk ( @{$dbChunkSet->get_all_DnaFragChunks()} ) {
+      push @db_chunk_files, $self->dumpChunkToTmp($db_chunk);
   }
 
   if (@db_chunk_files > 1) {
-    $self->warning("you have given a chunkset for the database; dumping individual chunks and creating a runnable for each one");
+    $self->warning("you have given a chunkset for the database; dumped individual chunks and creating a runnable for each one");
```

Comment:
This whole sentence is a little off. I am not sure what it is trying to say (not your fault, I don't think `dumping->dumped` improves much), but if you are going to change the word, depending on meaning:
"you have given a chunkset for the database; individual chunks have been dumped, creating a runnable ..."
"a chunkset has been provided (for or to?) the database, these have been dumped and runnables created per chunk"
Something along those lines/better.